databricks/koalas/base.py

```diff
@@ -323,3 +328,126 @@ def notnull(self):
+              original column labels.
+
+        >>> ks.Series([True, True]).all()
+        True
+
+        >>> ks.Series([True, False]).all()
+        False
+
+        >>> ks.Series([0, 1]).all()
+        False
+
+        >>> ks.Series([1, 2, 3]).all()
+        True
+
+        >>> ks.Series([True, True, None]).all()
+        True
+
+        >>> ks.Series([True, False, None]).all()
+        False
+
+        >>> ks.Series([]).all()
+        True
+
+        >>> ks.Series([np.nan]).all()
+        True
+        """
+
+        if axis not in [0, 'index']:
+            raise ValueError('axis should be either 0 or "index" currently.')
+
+        sdf = self._kdf._sdf.select(self._scol)
+        col = self._scol
+
+        # Note that we're ignoring `None`s here for now.
+        # any and every was added as of Spark 3.0
+        # ret = sdf.select(F.expr("every(CAST(`%s` AS BOOLEAN))" % sdf.columns[0])).collect()[0][0]
+        # Here we use min as its alternative:
+        ret = sdf.select(F.min(F.coalesce(col.cast('boolean'), F.lit(True)))).collect()[0][0]
+        if ret is None:
+            return True
+        else:
+            return ret
+
+
+    # TODO: axis, skipna, and many arguments should be implemented.
+    def any(self, axis: int = 0) -> bool:
+        """
+        Return whether any element is True.
+
+        Returns False unless there at least one element within a series that is
+        True or equivalent (e.g. non-zero or non-empty).
+
+        Parameters
+        ----------
+        axis : {0 or 'index'}, default 0
+            Indicate which axis or axes should be reduced.
+
+            * 0 / 'index' : reduce the index, return a Series whose index is the
+              original column labels.
+
+        Examples
+        --------
+
+        **Series**
+
+        For Series input, the output is a scalar indicating whether any element
+        is True.
+
+        >>> ks.Series([False, False]).any()
+        False
+
+        >>> ks.Series([True, False]).any()
+        True
+
+        >>> ks.Series([0, 0]).any()
+        False
+
+        >>> ks.Series([0, 1, 2]).any()
+        True
+
+        >>> ks.Series([False, False, None]).any()
+        False
+
+        >>> ks.Series([True, False, None]).any()
+        True
+
+        >>> ks.Series([]).any()
+        False
+
+        >>> ks.Series([np.nan]).any()
+        False
+        """
+
+        if axis not in [0, 'index']:
+            raise ValueError('axis should be either 0 or "index" currently.')
+
+        sdf = self._kdf._sdf.select(self._scol)
+        col = self._scol
+
+        # Note that we're ignoring `None`s here for now.
+        # any and every was added as of Spark 3.0
+        # ret = sdf.select(F.expr("any(CAST(`%s` AS BOOLEAN))" % sdf.columns[0])).collect()[0][0]
+        # Here we use max as its alternative:
+        ret = sdf.select(F.max(F.coalesce(col.cast('boolean'), F.lit(False)))).collect()[0][0]
```

Comment:
I switched previous ones to max. Seems it's simpler.

Reply:
@hvanhovell, does this make sense?

Reply:
How should we handle empty datasets? Should that return `False` instead of `None`?

Reply:
Ah, that's handled below if/else :).